setup.cfg

```diff
@@ -1,7 +1,7 @@
 [metadata]
 # replace with your username:
 name = luna-fviz
-version = 0.5
+version = 0.6
```

Comment:
We should not change this with every PR. We do not know which PR will be merged first. I would say we change this only once we want to release a new version.